fluent-syntax/src/ftlstream.js

```diff
@@ -208,7 +195,7 @@ export class FTLParserStream extends ParserStream {
   }
 
   skipToNextEntryStart() {
-    while (this.next()) {
+    while (this.ch) {
```

Comment:
When the syntax error happens exactly on the newline character, `this.next()` would overshoot the beginning of the entry on the next line, if present.